src/components/common/Accordion.js

```diff
@@ -29,6 +29,8 @@ const Accordion = ({ className, trigger, children }) => {
     }, [open])
 
     const handleClick = () => {
+        action && action()
```

Comment:
@marcinbodnar can we revert the changes to `Accordion`?

Reply:
@Patrick1904 I left it on purpose, it might be handy in the future to handle action by this component